### PR TITLE
fix: log 401 errors

### LINKED ISSUE #56: Log 402 token errors as error level
Currently the 401 token authorization errors seem to not be logged in an error level and thus are harder to identify.
Those are potential errors and typically should not happen if the app is handling it well. 
To identify issues better we should be informed about such failures.

### PR DESCRIPTION
**Pull request by im-adithya**
Fixes #56 and #68

### CODE REVIEW

**Comment by kiwiidb** [comment]
Tests seem to be failing but it seems like that's unrelated to this commit? 🤔 can you check why they are failing?

**Comment by bumi** [comment]
Those errors are strange. it was green and then red again. something else must be off there.

File: controllers/controllers.go [hunk 1]
```diff
@@ -63,8 +63,52 @@ func (ctrl *OAuthController) EndpointHandler(w http.ResponseWriter, r *http.Requ
 	}
 }
 
+func (ctrl *OAuthController) tokenError(w http.ResponseWriter, err error) error {
+	data, statusCode, header := ctrl.Service.OauthServer.GetErrorData(err)
+	logrus.Errorf("%s: %s", data["error_description"], data["error"])
+	sentry.CaptureException(err)
```
Comment (im-adithya) [inline controllers/controllers.go:69]: remove sentry for 401
(also https://github.com/getAlby/oauth2server/pull/61/files#diff-2873f79a86c0d8b3335cd7731b0ecf7dd4301eb19a82ef7a1cba7589b5252261L40)

**Comment by im-adithya** [comment]
Closing in favour of #74

**Pull request closed without merging**